libs/auth/src/java_card_config.ts

```diff
@@ -15,36 +18,62 @@ export interface JavaCardConfig {
   vxCertAuthorityCertPath: string;
 }
 
+/**
+ * Should be kept in sync with vxsuite-complete-system scripts
+ */
+const PROD_CONFIG_DIRECTORY = '/vx/config';
+
 /**
  * The password for all dev private keys
  */
 export const DEV_PRIVATE_KEY_PASSWORD = '1234';
 
-function constructDevCardProgrammingConfig(
-  pathToAuthLibRoot: string
-): JavaCardConfig['cardProgrammingConfig'] {
+function shouldUseProdCerts(): boolean {
+  return process.env.NODE_ENV === 'production' && !isVxDev();
+}
+
+function constructCardProgrammingConfig(): JavaCardConfig['cardProgrammingConfig'] {
+  if (shouldUseProdCerts()) {
+    assert(process.env.VX_ADMIN_PRIVATE_KEY_PASSWORD !== undefined);
+    return {
+      vxAdminCertAuthorityCertPath: path.join(
+        PROD_CONFIG_DIRECTORY,
+        'vx-admin-cert-authority-cert.pem'
+      ),
+      vxAdminPrivateKeyPassword: process.env.VX_ADMIN_PRIVATE_KEY_PASSWORD,
+      vxAdminPrivateKeyPath: path.join(
+        PROD_CONFIG_DIRECTORY,
+        'vx-admin-private-key.pem'
+      ),
+    };
+  }
   return {
-    vxAdminCertAuthorityCertPath: `${pathToAuthLibRoot}/certs/dev/vx-admin-cert-authority-cert.pem`,
-    vxAdminOpensslConfigPath: `${pathToAuthLibRoot}/certs/openssl.cnf`,
+    vxAdminCertAuthorityCertPath: path.join(
+      __dirname,
+      '../certs/dev/vx-admin-cert-authority-cert.pem'
+    ),
     vxAdminPrivateKeyPassword: DEV_PRIVATE_KEY_PASSWORD,
-    vxAdminPrivateKeyPath: `${pathToAuthLibRoot}/certs/dev/vx-admin-private-key.pem`,
+    vxAdminPrivateKeyPath: path.join(
+      __dirname,
+      '../certs/dev/vx-admin-private-key.pem'
+    ),
   };
```

Comment:
As you can sort of see here, my plan for prod is to have `vxsuite-complete-system` machine configuration scripts create and store the VxAdmin cert authority cert at `/vx/config/vx-admin-cert-authority-cert.pem`. Until we're using the TPM for signing, the scripts can also store the VxAdmin private key at `/vx/config/vx-admin-private-key.pem` and provide the private key password through the same env var mechanism that we use to provide [machine ID](https://github.com/votingworks/vxsuite-complete-system/blob/7226af99dfc1a552036f35e2e769fd95786abb93/config/read-vx-machine-config.sh#L36-L39).

Dev (including VxDev) will continue to use the dev keys and certs committed to the `certs/dev` directory.